wellcomeml/ml/sent2vec_vectorizer.py

```diff
@@ -3,21 +3,24 @@
 paper and codebase. https://github.com/epfml/sent2vec
 """
 from sklearn.base import TransformerMixin, BaseEstimator
-try:
-    import sent2vec
-except ImportError:
-    from wellcomeml.__main__ import download
-    download("non_pypi_packages")
-    import sent2vec
 
 from wellcomeml.utils import check_cache_and_download
 
 
 class Sent2VecVectorizer(BaseEstimator, TransformerMixin):
+
     def __init__(self, pretrained=None):
         self.pretrained=pretrained
 
     def fit(self, *_):
+
+        try:
```

Comment:
why inside fit and not outside? what happens if a user pickles the model and then loads and runs transform? i suspect that sent2vec will not be imported and it will fail, if I am right it is worth adding a test

Reply:
it failed the tests when it was in `__init__` (sent2vec not found). At the moment it looks like you can't load a model in this class anyway, so they'd have to fit before transforming anyway. 

I think that if a `load_model` function was added to this class, this is where the import sent2vec lines would live.

Reply:
i am worried that when the class is used after pickling it will fail. i was suggesting that you try to add it before init as you have done everywhere else. let me give it a try to see if fails

Reply:
actually, pickling fails atm anyway so no need to worry about that